### ISSUE TITLE
vimtex with latexmk does not respect provided options

### ISSUE THREAD

**Issue by jayeshbadwaik** [lervag/vimtex#771]
**UPDATE** : The error occurs when going between commits [7f7a319](https://github.com/lervag/vimtex/commits/c46debececdff6c2eda4da6a1e2dcff5d7dae837) and [c46debe](https://github.com/lervag/vimtex/commits/c46debececdff6c2eda4da6a1e2dcff5d7dae837). The test case works fine at and before 7f73a319. 

I am having these problems only recently, after updating to the latest version of Vimtex. The old version was probably around 3 months old). I am currently bisecting commit history to find out wherein the problem crept in. I will keep you updated on it. 

I use the shown vimrc file and I execute the VimtexCompile command to get my livepreview, whence, there is a mismatch between the options provided to vim and the options used by vimtex. 

The ouput of the command ```:echo g:vimtex_latexmk_options``` is 
```-lualatex -silent  -synctex=1 -interaction=nonstopmode```

and the output of the command 
```:VimtexInfo``` is 
```
 compiler: latexmk
    backend: jobs
    output: /tmp/vsIY00u/2
    configuration:
      continuous: 1
      callback: 1
      latexmk options:
        -verbose
        -pdf
        -file-line-error
        -synctex=1
        -interaction=nonstopmode    
```

which is not the same. 

### Minimal vimrc file

Please provide a minimal vimrc file that reproduces the issue. The following
should often suffice:

```vim
set nocompatible

" Load Vimtex
let &rtp  = '~/.vim/plugged/vimtex,' . &rtp
let &rtp .= '~/.vim/plugged/vimtex/after'

filetype plugin indent on
syntax enable

" Local Leader
:let maplocalleader = "\\"
:let g:vimtex_view_method = 'zathura'
"<" Options to latexmk
let g:vimtex_latexmk_options = '-lualatex -silent  -synctex=1
      \ -interaction=nonstopmode'
" We build our files in the build directory, which we then ignore
" this allows a much better view of the system.
let g:vimtex_latexmk_build_dir = 'livepreview'
let g:vimtex_quickfix_mode = 0


```

**Comment by lervag** [comment]
There's been some pretty large updates, in particular to the implementation of the compiler interface (i.e. `latexmk`). You should get the expected behaviour with:

```vim
let g:vimtex_compiler_latexmk = {
      \ 'options' : [
      \   '-lualatex',
      \   '-silent',
      \   '-synctex=1,
      \   '-interaction=nonstopmode',
      \ ],
      \ 'build_dir' : 'livepreview',
      \}
```

I think this should have been understandable if you had tried `:h g:vimtex_latexmk_options`, since it would show a relevant changelog entry that gives a brief explanation. However, I understand that it is not the first thing that comes to mind to check the documentation for why something stops working.

**Comment by jayeshbadwaik** [comment]
I did read it, but I saw that the options were deprecated, so did not pay much attention since I assumed I assumed that the options should work anyway and only will be removed in the future. However, I got my setup working with the new options. 

Thank you for the comment. There is `'` missing at the end of synctex link in your above comment (for future, if someone looks at it.) Anyway, cheers! and thanks for the plugin!
